acceptance/suites/pre_suite/foss/00_setup_environment.rb

```diff
@@ -1,2 +1,13 @@
-step "Initialize Test Config"
+step "Initialize Test Config" do
   PuppetServerExtensions.initialize_config options
+
+  if PuppetServerExtensions.config[:puppet_build_version] == 'LATEST'
```

Comment:
In order for these new settings to take effect in place of the ones that we're currently using in the [helper.rb file](https://github.com/puppetlabs/puppetserver/blob/bd05e6ef2ca0229c82526928843085bd50f0e58f/acceptance/lib/helper.rb#L26-L44), should this PR also set the corresponding helper.rb versions to "LATEST"?  Maybe it would also be worthwhile to have a value like `nil` be treated like `LATEST` for each setting?